src/context/cart-context.js

```diff
@@ -0,0 +1,65 @@
+import { useContext, createContext, useReducer } from "react";
+
+const CartContext = createContext(null);
+
+const cartReducerFunc = (cartState, action) => {
+  switch (action.type) {
+    case "ADD_TO_CART":
```

Comment:
Move this in to constant.js file and import wherever you need

Reply:
yes will do that at end now I will show them here because its good for readability I am working on multiple files so swapping it is time consuming but will do that at the end